org.eclipse.xtext.tests/src/org/eclipse/xtext/util/MergeableManifest2Test.java

```diff
@@ -971,6 +978,50 @@ public void hachCode_differ_becauseOfEntry() throws Exception {
 		assertNotEquals(manifest0.hashCode(), manifest1.hashCode());
 	}
 
+	@Test
+	public void readWrite_realisticManifest() throws Exception {
+		// @formatter:off
+		String content =
```

Comment:
Should we try to read all Manifest.MF resources from the classpath and check if they yield errors?

Reply:
would be at least a good manual test